examples/powertools-examples-core/README.md

```diff
@@ -23,6 +24,15 @@ For each of the tools, the example application is the same, and consists of the
 
 Configuration files and deployment process for each tool are described in corresponding README files.
 
+## Kotlin
+For each of the tools, the example application is the same, and consists of the following files:
```

Comment:
```suggestion
You can also use Kotlin! The example consists of the following files:
```

Reply:
I think the headings need a rejig - e.g the examples at the top are all java, and then we have the one Kotlin one. 

```md
# Examples
## 1. Java Examples 
gradle, maven, etc.
## 2. Kotlin Examples
```
or something perhaps? It is a bit confusing now and gives me the sense that I could have all the permutations - e.g. CDK + Kotlin.

Reply:
Looks good 👍